core/pom.xml

```diff
@@ -40,6 +40,11 @@
       <classifier>${avro.mapred.classifier}</classifier>
     </dependency>
     <dependency>
+      <groupId>com.amazonaws</groupId>
+      <artifactId>aws-java-sdk</artifactId>
+      <version>${aws.java.sdk.version}</version>
+    </dependency>
+    <dependency>
```

Comment:
I took a quick look and unfortunately this has more than 50 transitive dependencies (jackson, joda time, apache http client) that are likely to cause conflicts. I don't think we can merge this until we look into this more deeply. Can we use a more narrow version, for instance only the s3 sdk? Even then we'll still have many potential conflicts but it would at least reduce the amount of auditing we need to do.